### PR TITLE
Valid ip or host

### PR DESCRIPTION
**Pull request by adrian3ka**

### CODE REVIEW

File: test/validators/hostname.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import {expect} from 'chai';
+import validator from '../../lib';
+
+describe('IP address or Host validator', () => {
```
Comment (sendyhalim) [inline test/validators/hostname.js]: This should be `Hostname validator`

**Comment by febrianyuwono** [comment]
please update the `src/locale/id.js` too for localization

**Comment by adrian3ka** [comment]
> please update the `src/locale/id.js` too for localization

updated

File: test/validators/hostname.js (hunk 1 above)
Comment (adrian3ka) [reply test/validators/hostname.js]: done

File: src/locale/id.js [hunk 2]
```diff
@@ -54,5 +54,6 @@ module.exports = {
   'taxId:$1': 'Nomor NPWP tidak valid.',
   'url': 'Format link tidak valid.',
   'uuid:$1': 'Input harus berupa UUID.',
-  'validZipCode': 'Kode Pos tidak valid.'
+  'validZipCode': 'Kode Pos tidak valid.',
+  'hostname': 'Harus merupakan nama host yang valid'
```
Comment (sendyhalim) [inline src/locale/id.js]: `Nama host tidak valid`

File: src/index.js [hunk 3]
```diff
@@ -163,6 +165,7 @@ let validationMessages = {
   'requiredIf:$1:$2': requiredIf.message,
   'taxId:$1': taxId.message,
   'multipleOf:$1': multipleOf.message,
+  'hostname': hostname.message,
```
Comment (febrianyuwono) [inline src/index.js]: CMIIW, but shouldn't the `'hostname'` be put at line 181? after `fqdn`, and it doesn't need `'`

File: src/index.js [hunk 4]
```diff
@@ -101,6 +102,7 @@ let validation = {
   'requiredIf:$1:$2': requiredIf.validate,
   'taxId:$1': taxId.validate,
   'multipleOf:$1': multipleOf.validate,
+  'hostname': hostname.validate,
```
Comment (febrianyuwono) [inline src/index.js]: same as below

File: src/index.js (hunk 3 above)
Comment (adrian3ka) [reply src/index.js]: oops sorry i didn't realize that its ordered

**Pull request merged**